src/app/libp2p_helper/src/libp2p_helper/main.go

```diff
@@ -141,6 +146,23 @@ func main() {
 	decoder := capnp.NewDecoder(os.Stdin)
 
 	app := newApp()
+	host, err := libp2p.New()
+	if err != nil {
+		helperLog.Fatalf("failed to create libp2p host: %v", err)
+	}
```

Comment:
It's wrong that we have this code change.

Bitswap initialization should happen upon a `Configure` RPC is received by the libp2p helper.

See `src/app/libp2p_helper/src/codanet.go`, function `MakeHelper`.

If this code was required for tests to pass, it's likely that `Configure` wasn't called or something along these lines.